components/widgets/menu/menu_group.tsx

```diff
@@ -11,10 +11,16 @@ type Props = {
 }
 
 export default class MenuGroup extends React.PureComponent<Props> {
+    handleDividerClick = (e: React.MouseEvent) => {
+        e.preventDefault();
+        e.stopPropagation();
+    }
+
     public render() {
         const {children} = this.props;
 
-        const divider = this.props.divider || <li className='MenuGroup menu-divider'/>;
+        const divider = this.props.divider || <li className='MenuGroup menu-divider'
```

Comment:
I don't know if maybe here is better to add an `onClick` to all the dividers even the custom ones. I didn't do it because maybe the user wanted to have some custom behaviour